engine/src/test/scala/cromwell/SubFunctionWorkflowSpec.scala

```diff
@@ -0,0 +1,59 @@
+package cromwell
```

Comment:
@geoffjentry is centaur ready-enough for the prime-time for me to say `add this in centaur instead`?